source/TLS.rst

```diff
@@ -0,0 +1,123 @@
+Transport Layer Security (TLS)
+******************************
+
+Security is configured at multiple levels. Transport Layer Security
+(TLS) secures the API endpoints and is a
+cryptographic protocol. TLS provides secure communications over a computer
+network.
+
+The Percona Kubernetes Operator for PXC uses a cert-manager and supports manual configuration, which is available for all versions of K8s and is an upstream feature. A cert-manager is a Kubernetes tool widely used for to automate the management and issuance of TLS certificates.
```

Comment:
```suggestion
The Percona Kubernetes Operator for PXC can use a cert-manager for automatic certificates generation and supports also manual certificates generation. A cert-manager is a Kubernetes certificate management controller which widely used to automate the management and issuance of TLS certificates.
```

Reply:
I didn't see any change of wording in the suggested change. I removed the extra "for" in the line 9.

Reply:
The current wording is not very precise technically.